index.html

```diff
@@ -253,5 +261,20 @@ <h2>Skills</h2>
         </ul>
       </div>
     </section>
+    <!--contact-->
+    <section class="contact-section">
+      <div class="contact-container">
+        <div class="contact-div">
+          <h1>Contact me</h1>
+          <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          <form class="contact-form" action="" method="post">
```

Comment:
- [ ] Please form the project requirements, you should collect data by using the [Formspree service](https://formspree.io/html).
  - Using formspree, Just HTML forms. No Javascript, PHP, or servers are required to collect data for users.
  - After using formspree your form tag should looks like this :point_down: 
    - `<form action="https://formspree.io/f/{form_id}" method="post">`